packages/edge-config/src/index.ts

```diff
@@ -164,7 +164,12 @@ export function createClient(
 
   const baseUrl = connection.baseUrl;
   const version = connection.version; // version of the edge config read access api we talk to
-  const headers = { Authorization: `Bearer ${connection.token}` };
+  const headers: Record<string, string> = {
+    Authorization: `Bearer ${connection.token}`,
+  };
+
+  if (typeof process !== 'undefined' && process.env.NODE_ENV)
```

Comment:
Im wondering if VERCEL_ENV would be a better candidate since NODE_ENV is often set to production even when working locally

Reply:
Good point, makes sense. NODE_ENV is probably still more common if someone wouldn't use `vc env pull` etc. but VERCEL_ENV should probably provide the same insights.